api/python/provisioner/commands/deploy_vm.py

```diff
@@ -93,6 +93,38 @@ class DeployVM(Deploy):
     input_type: Type[inputs.NoParams] = inputs.NoParams
     _run_args_type = run_args_type
 
+    def set_pillar_data(self, setup_type):
+        minions_ids = ['srvnode-1']
```

Comment:
please add some newlines to improve readability

Reply:
comment is addressed